src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift

```diff
@@ -878,6 +879,41 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CoronaWarnAppDelegate, Re
 		}
 	}
 	
+	private func setupSelectedRegionsForTesting() {
```

Comment:
Hmm seems like our already big Appdelegate will grow and grow if we countinue to add more testing data. 
We could move all of the *ForTesting methods all together into there own extension of Appdelegate or one extension for each *ForTesting method.